test/e2e/config/docker.yaml

```diff
@@ -187,6 +187,7 @@ variables:
 intervals:
   default/wait-controllers: ["3m", "10s"]
   default/wait-cluster: ["5m", "10s"]
+  default/wait-cluster-infrastructureref: ["5m", "10s"]
```

Comment:
Let's use configurable timeout only for operations that might have a different duration on a different infrastructure.
In this case you can use a const timeout; also, I was expecting this timeout to be shorted (few seconds,  30s or 1m if we want to stay on the side)